src/controllers/userController.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/userService');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { id } = req.params;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const update = (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
+    res.sendStatus(422);
+
+    return;
+  }
+
+  const updatedUser = userService.updateUser({ id: +id, name });
+
+  if (!updatedUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(updatedUser);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+  }
```

Comment:
You should return after sending a response with status code 400, to prevent the execution of the remaining part of the function. If you don't return, the function will continue to execute and cause an error when you try to send the response again.